homeassistant/components/meater/config_flow.py

```diff
@@ -0,0 +1,66 @@
+"""Config flow for Meater."""
+from meater import AuthenticationError, MeaterApi, ServiceUnavailableError
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+from homeassistant.helpers import aiohttp_client
+
+# pylint: disable=unused-import
+from .const import DOMAIN
+
+
+class MeaterConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Meater Config Flow."""
+
+    async def async_step_user(self, user_input=None):
+        """Define the login user step."""
+        if user_input is None:
+            return self.async_show_form(
+                step_id="user",
+                data_schema=vol.Schema(
+                    {vol.Required(CONF_USERNAME): str, vol.Required(CONF_PASSWORD): str}
+                ),
```

Comment:
The schema is defined multiple times in the configuration flow, maybe put it into a constant for reuse instead?

Reply:
Fixed in 143f8b4